Bottom: fcd9031ab2321dda4ca83f9fe430e4f902558fe8
Top:    66e27229dc34ee55046e2b53c6250b8ad823f77a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-21 18:15:03 +0000

Refresh of add-cache-for-noholes-diced-ve

---

diff --git a/src/draw.c b/src/draw.c
index b1dd7ce..9c6d8ee 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -99,7 +99,7 @@ static void DrawPadLowLevel (hidGC, PadTypePtr, Boolean, Boolean);
 static void DrawPadNameLowLevel (PadTypePtr);
 static void DrawLineLowLevel (LineTypePtr, Boolean);
 static void DrawRegularText (LayerTypePtr, TextTypePtr, int);
-static void DrawPolygonLowLevel (PolygonTypePtr, void *);
+static void DrawPolygonLowLevel (PolygonTypePtr);
 static void DrawArcLowLevel (ArcTypePtr);
 static void DrawElementPackageLowLevel (ElementTypePtr Element, int);
 static void DrawPlainPolygon (LayerTypePtr Layer, PolygonTypePtr Polygon);
@@ -1726,7 +1726,7 @@ DrawTextLowLevel (TextTypePtr Text, int min_line_width)
  * lowlevel drawing routine for polygons
  */
 static void
-DrawPolygonLowLevel (PolygonTypePtr Polygon, void *data)
+DrawPolygonLowLevel (PolygonTypePtr Polygon)
 {
   if (!Polygon->Clipped)
     return;
@@ -2081,7 +2081,7 @@ DrawPolygon (LayerTypePtr Layer, PolygonTypePtr Polygon, int unused)
   else
     gui->set_color (Output.fgGC, Layer->Color);
   layernum = GetLayerNumber (PCB->Data, Layer);
-  DrawPolygonLowLevel (Polygon, NULL);
+  DrawPolygonLowLevel (Polygon);
   if (TEST_FLAG (CLEARPOLYFLAG, Polygon))
     {
       r_search (PCB->Data->pin_tree, &Polygon->BoundingBox, NULL,
@@ -2426,7 +2426,7 @@ ErasePolygon (PolygonTypePtr Polygon)
 {
   Erasing++;
   gui->set_color (Output.fgGC, Settings.BackgroundColor);
-  DrawPolygonLowLevel (Polygon, NULL);
+  DrawPolygonLowLevel (Polygon);
   Erasing--;
 }
